romanisim/image.py

```diff
@@ -217,33 +214,38 @@ def trim_objlist(objlist, image):
 
 
 def add_objects_to_image(image, objlist, xpos, ypos, psf,
-                         flux_to_counts_factor, bandpass=None, filter_name=None,
-                         rng=None, seed=None):
+                         flux_to_counts_factor, exptimes=None,
```

Comment:
A few thoughts on this function:
- You've looked more closely than I have, but there's some kind of ~duplication or redundancy about exptimes and flux_to_counts_factor.  For L2 operation, you just need one of these.  For L3 operation, we scale the flux_to_counts_factor by exptime and then divide it out by exptime later.  I can imagine something like scale_exptime instead, that defaults to 1, and multiplies flux_to_counts_factor and is also used as a denominator.  Not needed as a change, but worth thinking about.
- If we kept the API to only support galsim objects, would that make the caller significantly more complicated?  That would make the API simpler and not need to have the WCS come in in two places.

Reply:
I separated them to allow for unit change, if desired.. I could rename them to make that more clear?

We can have it only support galsim objects, it just requires making one of the base mosaic that objects are being written to within the L3 code - I can make that change.

Reply:
Conversion to image implemented.

Reply:
I didn't rename the variables, though I did include the unit scaling you mentioned below. I am willing to discuss the variable names /usage further, if you desire.

Reply:
I renamed one of the variables as part of a change below.